backend/src/data_entry/structs.rs

```diff
@@ -224,33 +222,29 @@ pub struct DifferencesCounts {
     /// Number of fewer counted ballots ("Er zijn minder stembiljetten geteld. Hoeveel stembiljetten zijn er minder geteld")
     #[schema(value_type = u32)]
     pub fewer_ballots_count: Count,
-    /// Number of unreturned ballots ("Hoe vaak heeft een kiezer het stembiljet niet ingeleverd?")
-    #[schema(value_type = u32)]
-    pub unreturned_ballots_count: Count,
-    /// Number of fewer ballots handed out ("Hoe vaak is er een stembiljet te weinig uitgereikt?")
-    #[schema(value_type = u32)]
-    pub too_few_ballots_handed_out_count: Count,
-    /// Number of more ballots handed out ("Hoe vaak is er een stembiljet te veel uitgereikt?")
-    #[schema(value_type = u32)]
-    pub too_many_ballots_handed_out_count: Count,
-    /// Number of other explanations ("Hoe vaak is er een andere verklaring voor het verschil?")
-    #[schema(value_type = u32)]
-    pub other_explanation_count: Count,
-    /// Number of no explanations ("Hoe vaak is er geen verklaring voor het verschil?")
-    #[schema(value_type = u32)]
-    pub no_explanation_count: Count,
+    /// Whether total of admitted voters and total of votes cast match.
+    /// ("D en H zijn gelijk")
+    pub admitted_voters_equals_votes_cast: bool,
+    /// Whether total of admitted voters is greater than total of votes cast match.
+    /// ("H is groter dan D (meer uitgebrachte stemmen dan toegelaten kiezers)")
+    pub votes_cast_greater_than_admitted_voters: bool,
+    /// Whether total of admitted voters is less than total of votes cast match.
+    /// ("H is kleiner dan D (minder uitgebrachte stemmen dan toegelaten kiezers)")
+    pub votes_cast_smaller_than_admitted_voters: bool,
```

Comment:
To make it more clear that this is one question, I would move these three booleans to their own struct and use a field `is_admitted_voters_equal_to_votes_cast` in `DifferencesCounts` to refer to that new struct.